src/test/compile-fail/deriving-meta-unknown-trait.rs

```diff
@@ -11,7 +11,7 @@
 // ignore-tidy-linelength
 
 #[derive(Eqr)]
-//~^ ERROR cannot find derive macro `Eqr` in this scope
+//~^ ERROR cannot find macro `Eqr` in this scope
```

Comment:
I think attribute macros should say ``"cannot find attribute macro `...` in this scope"``.